source/core/owt_base/InternalOut.cpp

```diff
@@ -33,15 +33,15 @@ void InternalOut::onFrame(const Frame& frame)
     m_transport->sendData(sendBuffer, header_len + 1, reinterpret_cast<char*>(const_cast<uint8_t*>(frame.payload)), frame.length);
 }
 
-void InternalOut::onFrame(uint8_t *buffer, uint32_t length)
+void InternalOut::onFrame(uint8_t *buffer, int width, int height, uint32_t length)
```

Comment:
The internalOut already has a "onFrame(const Frame&)" function, why you add a function and construct frame struct inside?
Why not construct a frame struct directly and call the previouse "onFrame" function?
Like "x->onFrame(outFrame)"